roles/blackbox_exporter/defaults/main.yml

```
@@ -65,3 +65,6 @@ blackbox_exporter_configuration_modules:
 #      preferred_ip_protocol: ip6
 #      validate_answer_rrs:
 #        fail_if_matches_regexp: [test]
+
+# Where to put the blackbox_exporter.yml main configuration file
+blackbox_exporter_conf_dir: /etc
```

Let's be consistent with `conf` vs `config` on all roles.

```suggestion
blackbox_exporter_config_dir: /etc
```

---

Changes applied. Apologies for not knowing the purpose of the meta/argument_specs.yml file.